### ISSUE TITLE
gogglesmm.desktop problems.

### ISSUE THREAD

**Issue by C0RD** [gogglesmm/gogglesmm#117]
During packaging for Debian I came over some missing or inconsistent information in gogglesmm.desktop 

This .desktop file is either missing a Keywords entry, or it does not contain keywords above and beyond those already present in the Name or GenericName entries.
https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s06.html,
https://wiki.gnome.org/Initiatives/GnomeGoals/DesktopFileKeywords for details.

**Comment by gogglesguy** [comment]
So what's the issue here?

https://github.com/gogglesmm/gogglesmm/blob/59756fdd4c4d8845e71205afe557ede86605b23f/extra/gogglesmm.desktop#L1-L20

According to the posted spec, Keywords is clearly marked as an optional item.

**Comment by C0RD** [comment]
Yes, you are right. please consider this as a suggestion then. The 'Linter' for Debian Packages reports this. 

There is two other warnings (and three hints) regarding extras/gogglesmm.appdata.xml at https://appstream.debian.org/sid/main/issues/gogglesmm.html :

- The component ID is not a reverse domain-name.
- A screenshot must contain at least one image or video in order to be useful

thank you, Cord
